src/components/NoListingImagePreview/NoListingImagePreview.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { AspectRatioWrapper } from '../../components';
+import { colorSchemes } from '../../util/types';
+
+import css from './NoListingImagePreview.module.css';
+
+/**
+ * NoListingImagePreview
+ * Component for listings without an image.
+ * Renders the listing title inside a styled, fixed-aspect-ratio container.
+ * @component
+ * @param {Object} props
+ * @param {string} props.style color scheme to apply
+ * @param {string} props.listingTitle listing title to display
+ * @param {number} props.width aspect ratio width
+ * @param {number} props.height aspect ratio height
+ * @param {Function?} props.setActivePropsMaybe optional hover handlers for map highlighting
+ * @param {string?} props.className optional CSS class for outer wrapper
+ * @returns {JSX.Element} component to display when no listing image is available
+ */
+const NoListingImagePreview = props => {
```

Comment:
I think we should rename this component to something else. This is an alternative to listing images, but it feels strange to name it as a negation of having listing image.

So, I was first thinking about these: 
- DefaultCardStyle
- StyledListingCardGraphics
- StyledListingCardBackground
- ListingCardBackground

...but Console defines "Listing thumbnail aspect ratio", so perhaps this should be about thumbnail:
- DefaultListingThumbnail
- ListingCardThumbnail
- StyledListingCardThumbnail
- ?